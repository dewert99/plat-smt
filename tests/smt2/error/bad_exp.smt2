(assert (and ()))
(assert 0)
(assert #b010)
(assert #xA1eBadF00D)
(assert 0.3)