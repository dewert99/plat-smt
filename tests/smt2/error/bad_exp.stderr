unexpected token when parsing expression at 1:14
(assert (and ()))
             ^

unsupported arithmetic at 2:9
(assert 0)
        ^

unsupported bitvec at 3:9
(assert #b010)
        ^

unsupported bitvec at 4:9
(assert #xA1eBadF00D)
        ^

unsupported decimal at 5:9
(assert 0.3)
        ^

